packages/components/tests/spec/file-uploader_spec.js

```diff
@@ -53,16 +56,17 @@ describe('File Uploader', function() {
     it('should set default options', function() {
       expect(flattenOptions(instance.options)).toEqual({
         selectorInit: '[data-file]',
-        selectorInput: 'input[type="file"].bx--file-input',
+        selectorInput: `input[type="file"].${prefix}--file-input`,
```

Comment:
We don't do variable prefix in test at this moment, rather tests see if default prefix is applied.